frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+export enum ColorPill {
+  Light,
+  Dark,
+}
+</script>
+
+<script setup lang="ts">
+export interface Props {
+  icon: boolean;
+  color?: ColorPill;
```

Comment:
Tohle by mělo být spíš:

```suggestion
  active?: boolean;
```

Reply:
Změněno.